requirements/requirements.dev.txt

```diff
@@ -35,6 +35,7 @@ mccabe==0.6.1             # via flake8
 newrelic==2.106.0.87
 oauthlib==1.0.3
 pluggy==0.6.0             # via tox
+psycopg2==2.7.4
```

Comment:
Just curious, why add this now? Is there a reason it hasn't been included yet?

Reply:
that's auto-generated so I'm not sure why was suddenly included. I'll look into it.